endoreg_db/views/anonymization_overview.py

```
@@ -0,0 +1,67 @@
+    return Response({"detail": f"Anonymization started for {kind} file"})
+
+# ---------- validate ----------------------------------------------------
+@api_view(["POST"])
+@permission_classes(PERMS)
+def validate_anonymization(request, file_id: int):
+    kind = AnonymizationService.validate(file_id)
+    if not kind:
+        return Response({"detail": "File not found"}, status=status.HTTP_404_NOT_FOUND)
+    return Response({"detail": f"Anonymization validated for {kind} file"})
+
+@api_view(["GET"]
```

Pylint (3.3.7)

[error] Parsing failed: ''(' was never closed (endoreg_db.views.anonymization_overview, line 53)'

(E0001)